validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/graffiti/Bytes32Converter.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.validator.client.loader.graffiti;
+
+import java.nio.charset.StandardCharsets;
+import org.apache.tuweni.bytes.Bytes32;
+
+public class Bytes32Converter {
```

Comment:
(nit) I think I'd call this something like `GraffitiParser`.  We use "converters" in the CLI package because this is a [picocli concept](https://picocli.info/#_custom_type_converters).

Reply:
I think what I am trying to achieve here is separating the dirty business of converting from _a String of some length_ to a Bytes32 from the higher level concept of **Graffiti**.

And then "GraffitiParser" would ask this "Bytes32parsingconvertingthingy" to do most of the work for it. So maybe we need both names.

Does that make sense?

Reply:
You mean we'd have `GraffitiConverter`, `GraffitiParser`, and `Bytes32Parser` ?